cypress/integration/chi-custom-elements/button.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+
+describe('Buttons', () => {
+
+  it('Base button should be included inside the custom element', () => {
+    cy.visit('tests/custom-elements/buttons.html');
+
+    cy.get('[data-cy="button"]')
+      .wait(5000)
```

Comment:
I would prefer avoid the wait function. If you are waiting for buttons.html page, I would prefer to use:
```
cy.url()
        .should('include', 'test/custom-elements/buttons.html');
```

Reply:
@cristb `cy.url().should('include', 'test/custom-elements/buttons.html');` will work even though the chi-buttons are not fully rendered, If you check this page it takes time to fully render the chi-buttons that's why I'm using wait() here so as to perform tests after the buttons are fully rendered. Any thoughts?

Reply:
@srinath-koneru Maybe checking for the hydrated class to be set. That class means that the component has loaded and cypress will wait until it finds it. 

```
cy.get('[data-cy="button"]').should('have.class', 'hydrated')
```
if it times-out, try:
```
cy.get('[data-cy="button"]', { timeout: 10000 }).should('have.class', 'hydrated')
```
Documentation available: https://docs.cypress.io/api/commands/should.html#Timeouts

Reply:
This was the other solution but I went with wait(). Agree with the selectors though I'm already changing that.